recipes/refl-cpp/all/test_package/example.cpp

```diff
@@ -0,0 +1,5 @@
+#include "refl.hpp"
+
+int main() {
+    return 0;
```

Comment:
Can you please add a line to all a method or ctor to test the linkage (or make sure theres no missing deps)